.github/workflows/data-de_dresden_www.yml

```diff
@@ -0,0 +1,55 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Data Provider
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  schedule:
+    - cron:  '5 12 * * *'
+  push:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+     
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
```

Comment:
Maybe a second order optimization: if this is the name of a docket image, using alpine and friends could speed up things.